build/yaml/steps/microbuild/codesignverify.yaml

```diff
@@ -15,6 +15,6 @@ steps:
   displayName: Verify Signed Files
   condition: and(succeeded(), in(variables['SignType'], 'Real', 'Test'))
   inputs:
-    TargetFolder: $(Build.BinariesDirectory)
+    TargetFolder: $(Build.StagingDirectory)
```

Comment:
Long term we should consider building into the Binaries folder. Do our static analysis builds work properly here?

Reply:
Currently they're pointing to the BuildArtifactStagingDirectory as well (note that BuildArtifactStagingDirectory and StagingDirectoy are interchangeable).